### PR TITLE
Create travis config and eslint

### PR DESCRIPTION
**Pull request by birdbrained**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,4 +6,4 @@ install:
 script:
     - ember github-pages:commit --message "gh-pages release"
 after_success:
-    - 'HELLO'
+    - 'HELLdO'
```
Comment (cameronblandford) [inline .travis.yml]: HdEY THERE

File: .travis.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ install:
     - yarn install
     - bower install
 script:
-    - yarn test:cover
```
Comment (cameronblandford) [inline .travis.yml]: What's the motivation behind this??

File: app/models/collection.js [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
-    dateCreated: DS.attr('date'),
-    dateUpdated: DS.attr('date'),
-    settings: DS.attr('object'),
-    groups: DS.hasMany('group'),
-    items: DS.hasMany('items'),
+const {
+    Model,
+    attr,
+    hasMany,
+    blenogsTo,
```
Comment (cameronblandford) [inline app/models/collection.js]: Might wanna fix this guy!

File: app/models/group.js [hunk 4]
```diff
@@ -1,12 +1,19 @@
...
-    tags: DS.attr('string'),
-    createdBy: DS.belongsTo('user'),
-    dateCreated: DS.attr('date'),
-    dateUpdated: DS.attr('date'),
-    items: DS.hasMany('item'),
+const {
+    Model,
+    attr,
+    hasMany,
+    blenogsTo,
```
Comment (cameronblandford) [inline app/models/group.js]: and this guy

File: app/models/item.js [hunk 5]
```diff
@@ -1,15 +1,21 @@
...
-    collection: DS.belongsTo('collection'),
-    group: DS.belongsTo('group'),
-    createdBy: DS.belongsTo('user'),
-    metadata: DS.attr('string'),
-    dateAdded: DS.attr('date'),
-    dateUpdated: DS.attr('date'),
+const {
+    Model,
+    attr,
+    blenogsTo,
```
Comment (cameronblandford) [inline app/models/item.js]: etc

File: .travis.yml (hunk 2 above)
Comment (birdbrained) [inline .travis.yml]: We dont need test coverage right now. Our tests first off are basically nonexistant, and coverage isn't a priority rn.

File: .travis.yml (hunk 1 above)
Comment (birdbrained) [inline .travis.yml]: lol probably can clean that.

File: app/models/collection.js (hunk 3 above)
Comment (birdbrained) [inline app/models/collection.js]: ⚾️ ✋ 👍

File: .travis.yml (hunk 1 above)
Comment (cameronblandford) [inline .travis.yml]: haha i liked it

**Pull request merged**